Source/stores.cpp

```diff
@@ -692,33 +692,23 @@ void __fastcall PrintStoreItem(ItemStruct *x, int l, char iclr)
 
 void __cdecl S_StartSBuy()
 {
-	int v0;  // ST10_4
-	int v1;  // eax
-	int *v2; // ecx
+	int i;
 
-	v0 = plr[myplr]._pGold;
 	stextsize = 1;
 	stextscrl = 1;
 	stextsval = 0;
-	sprintf(tempstr, "I have these items for sale :           Your gold : %i", v0);
-	AddSText(0, 1, 1u, tempstr, COL_GOLD, 0);
+	sprintf(tempstr, "I have these items for sale :           Your gold : %i", plr[myplr]._pGold);
+	AddSText(0, 1, 1, tempstr, COL_GOLD, 0);
 	AddSLine(3);
 	AddSLine(21);
 	S_ScrollSBuy(stextsval);
-	AddSText(0, 22, 1u, "Back", COL_WHITE, 0);
+	AddSText(0, 22, 1, "Back", COL_WHITE, 0);

```

Comment:
To me it looks like the compiler decided to zero last parameter last, I think this falls in the category of min-diff that we haven't been able to solve previously.